.gitignore

```diff
@@ -1,6 +1,5 @@
 *.iml
-.clj-kondo/
-.lsp/
+.cache
```

Comment:
Both clj-kondo and clojure-lsp use a `.cache` directory, that this the only part that should be ignored, the rest is configuration.